src/Raven.Server/Utils/Cli/RavenCli.cs

```diff
@@ -1133,7 +1133,7 @@ private static bool CommandHelp(List<string> args, RavenCli cli)
                 new[] {"script <server|database> [database]", "Execute script on server or specified database. WARNING: Use with care!"},
                 new[] {"logout", "Logout (applicable only on piped connection)"},
                 new[] {"openBrowser", "Open the RavenDB Studio using the default browser"},
-                new[] {"resetServer", "Restarts the server (shutdown and re-run)"},
+                new[] {"restartServer", "Restarts the server (shutdown and re-run)"},
```

Comment:
You have to also have the old command functional.

Reply:
For back compatibility?

Reply:
yes

Reply:
So, I'll modify the implementation to accept `resetServer`
and what about the description? to describe the old version too? (same question about the documentation)

Reply:
Both should use the same description